user/user.proto

```diff
@@ -153,4 +153,20 @@ message ChangeEmailAddressRequest {
 message VerifyChangedEmailAddressRequest {
   Authentication auth = 1;
   string secret_code = 2;
+}
+
+// EchoService receive string and returning it with maybe more data
+service EchoService {
+  // Echo receive string and returning it with maybe more data
+  rpc Echo(EchoRequest) returns (EchoResponse) {}
+}
```

Comment:
Ini nangkepnya malah cuma kayak buat debug apakah grpc nya jalan atau engga. Memang tujuannya mau seperti itu, atau ada tujuan lain?

Kalau rencananya mau return GUID atau return lainnya, at least schemanya harus dibikin jelas, jangan punya request yang `string request` dan response yang `string response` doang.

Reply:
saat ini sih kepakenya cuma itu doang ya, jadi drop aja?

Reply:
Not so sure soal "drop aja", tapi kalo emang butuh temporary proto file buat initialize project (dan initialize endpoints), ya bisa aja. Presentation layer can always comes later.